### ISSUE TITLE
Book 000B2E85 title typo

### ISSUE THREAD

**Issue by Constellation-VASCO** [Starfield-Community-Patch/Starfield-Community-Patch#686]
Data slate in First Cav Mech Factory has a typo in its title

---

## Details 
When hunting down the first cavalry, there are a few data slates in the mech factory.

One of them, with FormID 000B2E85 has a typo in the english version.

The title reads "Message to StaffL Honored Guest"

I'm pretty sure that L was intended to be a colon, so it should probably read "Message to Staff: Honored Guest"

## Game Version
1.8.86.0

## Language
English (EN)

## New Game
No

## Reproduction Steps
 1. Look at 000B2E85 in Starfield.esm
 2. ???
 3. Profit

**Comment by Pickysaurus** [comment]
This form has reflection data so cannot be edited by xEdit yet!

**Comment by Constellation-VASCO** [comment]
Yeah, I discovered that lovely little problem myself when trying to incorporate it into my mod, too. :(

Quite a few random books and slates seem to be that way, with no rhyme or reason why.
